core/metadata-3.3.md

```diff
@@ -0,0 +1,56 @@
+---
+title: IRCv3.3 Metadata
+layout: spec
+copyrights:
+  -
+    name: "Kiyoshi Aman"
+    period: "2016"
+    email: "kiyoshi.aman@gmail.com"
+---
+## Introduction
+
+This specification incorporates the IRCv3.2 metadata specification and
+incorporates it by reference.
+
+## METADATA
+
+### METADATA PRIORITY
+
+This subcommand provides a mechanism for clients to inform the server what
+priorities to assign to particular keys. The default priority is 'on
+subscription' for all keys. The format of `METADATA PRIORITY` MUST be as
+follows:
+
+    METADATA <Scope> PRIORITY <Priority> :<Keys>
+
+`<Scope>` is one of `global`, `channel`, or `user`, reflecting all subscriptions,
+channel-related subscriptions, and subscriptions via `MONITOR`.
+
+`<Priority>` operates in descending order, with `sub` enabling on-subscription
+and on-change notifications, `change` permitting only on-change notifications,
+and `request` requiring that the client explicitly request metadata with the
+relevant keys.
```

Comment:
Can some example keys be provided where the different `PRIORITY` values would be used and useful?

I can't see the benefits of priorities. It seems something like Attila suggested above with just `SUB`, `UNSUB`, and `SUBS` (something to see current subscriptions) or similar would make more sense?